sdk/tests/conformance2/rendering/draw-buffers.html

```diff
@@ -375,31 +375,37 @@
 
   checkAttachmentsForColor(attachments, [0, 255, 0, 255]);
 
-  // GLES3 spec section 3.9.2 Shader Outputs
-  debug("test that gl_FragColor only writes to color number zero");
+  // By default, gl_FragColor only writes to color number zero.
```

Comment:
Why remove the original test of the default behavior? I think it's still valuable.